package.json

```diff
@@ -13,8 +13,7 @@
     "build": "run-p build:*",
     "build:esm": "tsc -p tsconfig.build.json -m es2015 --outDir ./dist/esm && cp -r ./src/assets/ ./dist/esm/assets",
     "build:cjs": "tsc -p tsconfig.build.json -m commonjs --outDir ./dist && cp -r ./src/assets/ ./dist/assets",
-    "postbuild": "cp package.json ./dist/ && yarn postbuild:write-svgs",
```

Comment:
We're no longer building the icons and exporting them as part of the package, so this can be removed